src/DynamoCore/Engine/EngineController.cs

```diff
@@ -177,7 +177,7 @@ public void EnableProfiling(bool enable, HomeWorkspaceModel workspace, IEnumerab
                 astBuilder.ProfilingSession = null;
             }
 
-            workspace.MarkNodesAsModifiedAndRequestRun(nodes);
+            workspace.MarkNodesAsModifiedAndRequestRun(nodes, true);
```

Comment:
We need to ensure we pass `true` to the second parameter (which otherwise defaults to `false`) since we need to force-execute the node again (make sure it actually runs a second time but this time with the timing turned on). If this node is not made to force execute, delta compute will just skip it as it has already been executed before. This might explain the zero time execution.